.github/workflows/custom_docker_builds.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           - docker-image: ./images/cache-indexer
             image-tags: ghcr.io/spack/cache-indexer:0.0.3
           - docker-image: ./analytics
-            image-tags: ghcr.io/spack/django:0.3.30
+            image-tags: ghcr.io/spack/django:0.3.31
```

Comment:
This is pretty minor but, since we're upgrading Django to 5.0, should we make this a minor version change?

Reply:
https://github.com/spack/spack-infrastructure/pull/1021/commits/8b66a8832f7ea74e2f2e407d4c4fd0b3fd827840

Reply:
> This is pretty minor

:drum:

Reply:
Completely unintentional believe it or not :laughing: